cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -35,6 +35,42 @@ describe("Tab management", () => {
         });
       });
     });
+
+    describe("Editing the time", () => {
+      beforeEach(() => {
+        cy.get('[data-cy="contract"]').click();
+      });
+
+      it("properly handles employment contract", () => {
+        cy.get('[data-cy="employment_contract"]').click();
+        cy.get('[data-cy="contract-time-dropdown"]').click().get('[data-cy="full"]').click();
```

Comment:
Sama zmiana wymiaru etatu jeszcze nic nie sprawdza. W tym momencie jedyna opcja żeby test nie przeszedł to gdy aplikacja się wysypie albo `[data-cy="contract-time-dropdown"]` nie będzie widoczne.
Dopisz sprawdzenie czy po zapisaniu workera (`[data-cy="btn-save-worker"]`) jest widoczny odpowiedni ekran (`cy.get('[data-cy="btn-add-worker"]').should("be.visible")`) i czy widać zmieniony etat.